editor/core/constants.py

```diff
@@ -34,7 +34,8 @@ def PAST_PROJECTS_PATH(user) -> epath.Path:
 METADATA = "Metadata"
 RESOURCES = "Resources"
 RECORD_SETS = "Record Sets"
-TABS = [OVERVIEW, METADATA, RESOURCES, RECORD_SETS]
+RAI = "Extension: Responsible AI"
+TABS = [OVERVIEW, METADATA, RESOURCES, RECORD_SETS, RAI]
```

Comment:
Not sure where the "Responsible AI" tab should go:

- at the end?
- in-between "Metadata" and "Resources"?